More detailed image_url information (#30009)
home-assistant/home-assistant.github.io: source/_integrations/msteams.markdown

| ---------------------- | -------- | ----------- |
| `image_url`            |      yes | Attach an image to the message.

> The image URL must start with http, and as outlined from Microsoft in the [Documentation](https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-reference#common-properties-for-all-cards), the picture must be on a publicly available location.

Reviewer: According to the documentation you referenced, it must be an HTTPS URL?

```suggestion
The image must be an HTTPS URL, and as outlined by Microsoft in the [Documentation](https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-reference#common-properties-for-all-cards), the picture must be on a publicly available location.
```
